cor_surrogates/code/.nfs0000000006dc75ee000066e8

```diff
@@ -0,0 +1,291 @@
+#-----------------------------------------------
```

Comment:
What’s up with this file name?

Reply:
👍 ya, for some reason this didn't get deleted right away. Had to log off rhino and log back in and it was magically gone.